account/src/main/java/com/caldevsupplychain/account/util/UserMapper.java

```diff
@@ -6,23 +6,28 @@
 import com.caldevsupplychain.account.model.User;
 import com.caldevsupplychain.account.vo.UserBean;
 import com.caldevsupplychain.common.ws.account.UserWS;
+import org.mapstruct.Named;
+import org.springframework.stereotype.Component;
 
 import java.util.List;
 
 @Mapper(config = MapperBaseConfig.class, uses = {RoleMapper.class})
+@Component
```

Comment:
I thought you mentioned `Mapper` acts as `Component` as well?

Reply:
Yes it does, and actually can delete @component but it does show an underline red text beneath the userMapper instance in Controller class, even though it doesn't affect the code at all but I just adding the @Component to make that red underline disappear because it doesn't look nice in the code.